vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -8693,6 +8693,15 @@ For an irrotational electric field, the voltage is independent of the path betwe
   rdfs:label "Voltage" ;
   skos:broader quantitykind:EnergyPerElectricCharge ;
 .
+quantitykind:VoltageLoss
```

Comment:
I don't think this is right. The dimension of Voltage Loss is the same as Voltage, namely ML2T-3I-1.

Reply:
I knew that you would take issue with this but I did find a reference to voltage loss for the unit that said that voltage loss is measured in ampere-squared-hour (or ampere-squared-second). Now I am looking for the reference. In the meantime, I found a couple of other references to it that are both similar and different:

https://grabcad.com/library/ampere-squared-hour-meter-1
BACnet - Ampere square hours (246) ???

The ampere-squared-seconds is a unit measure based on the energy or heat allowed to pass through the fuse or circuit breaker during a short-circuit condition.  The unit is derived by squaring the effective RMS current and multiplying it by the total time it takes the fuse or circuit breaker to open and extinguish the electrical arcing.  The total clearing ampere-squared-seconds (I²tc) is the sum of the “opening” ampere-squared-seconds (I²to) plus “arcing” ampere-squared-seconds (I²ta).  (I²tc = I²to + I²ta).

https://www.optifuse.com/glossary.php

The DV associated with VoltageLoss is simply the units for A2-HR, so if VoltageLoss is not the quantity kind for A2-HR then we fix that and we are ok.

Reply:
I suggest we put these questionable quantity kinds into a holding pen until we can bring on a subject matter expert to help us resolve them. My intuition tells me that the dimensionality should be the same as voltage, but I don't know what I don't know.